src/prisma/__mocks__/prisma.service.mock.ts

```diff
@@ -31,6 +33,16 @@ export class MockPrismaService {
     delete: function (_request: { id: number }): Promise<void> {
       return Promise.resolve();
     },
+
+    findUnique: function (request: { where: { id: number } }): Promise<User> {
+      userId = request.where.id;
+      return Promise.resolve({
+        id: userId,
+        email: faker.internet.email(),
+        password: faker.internet.password(),
+        role: 'USER',
```

Comment:
```suggestion
        role: Role.USER,
```
always use predefined variables if possible
(this will improve the readability of the code and make future refactoring easier)